internal/config/profile.go

```diff
@@ -225,18 +225,38 @@ func (p *profile) SetOrgID(v string) {
 	p.Set(orgID, v)
 }
 
+// GetConfigDescription returns a map describing the configuration
+func GetConfigDescription(name string) (map[string]string, error) {
```

Comment:
I think is safe now to remove the error form the return
```suggestion
func GetConfigDescription(name string) map[string]string {
```